apps/aecontract/src/aect_channel_contract.erl

```diff
@@ -145,7 +145,7 @@ make_call_def(OwnerPubKey, ContractPubKey, GasLimit, GasPrice, Amount,
     , tx_env          => OnChainEnv
     , off_chain       => true
     , on_chain_trees  => OnChainTrees
-    , origin          => OwnerPubKey
+    , origin          => CallerPubkey
```

Comment:
Is this always true? Looks strange...

Reply:
I'm unsure how calls from inside an contract are handled.

Reply:
I think it is only the top-level call that enters `aect_channel_contract:run` (there is only one place in the code where this function is called) so this is correct. But I will double check.